tock/hours/views.py

```diff
@@ -942,6 +943,14 @@ def get_queryset(self, queryset=None):
         sorted_fiscal_years = sorted(fiscal_years.items(), reverse=True)
         return sorted_fiscal_years
 
+    def get_context_data(self, **kwargs):
+        # Call the base implementation first to get a context
+        context = super().get_context_data(**kwargs)
+        today = localdate()
+        year = today.year + 2 if today.month >= 10 else today.year + 1
```

Comment:
this line could maybe use a comment explaining the logic behind it?

Reply:
The logic has changed, but I tried to put as much doc as I can.